iabgpp-encoder/src/main/java/com/iab/gpp/encoder/section/TcfCaV1.java

```diff
@@ -55,7 +55,7 @@ private void initFields() {
     fields.put(TcfCaV1Field.CONSENT_SCREEN, new EncodableFixedInteger(6, 0));
     fields.put(TcfCaV1Field.CONSENT_LANGUAGE, new EncodableFixedString(2, "EN"));
     fields.put(TcfCaV1Field.VENDOR_LIST_VERSION, new EncodableFixedInteger(12, 0));
-    fields.put(TcfCaV1Field.TCF_POLICY_VERSION, new EncodableFixedInteger(6, 2));
+    fields.put(TcfCaV1Field.TCF_POLICY_VERSION, new EncodableFixedInteger(6, 1));
```

Comment:
The Tcf canada policy version seems to be 2 according to this vendor list: https://vendor-list.consensu.org/v2/ca/vendor-list.json